File: node01/ex05.js
const arr1=['item1','item2',{key1:'val1'}];
let a=arr1[0];
let b=arr1[1];
let c=arr1[2];
console.log(a,b,c);
// let [d,e,f] =arr1;
// console.log(d,e,f);
let [d,,{key1}] =arr1;
console.log(d,key1);

const obj={'key11':'val1','key12':'val2'};
let {key11,key12}=obj;
console.log(key11,key12);

File: py01/ex05.py
set1={2,5,7,3}
set2=set(['item5','item6'])
set2.add('item1')
set2.add('item2')
set2.add('item3')
set2.add('item4')

set2.remove('item3')

arr1=list(set2)

print(type(set1),set1)
print(type(set2),set2)
print(type(arr1),arr1)
print('-'*40)
map1={'key1':'val1','key2':'val2'}
map2=dict()
map2['key1']='val1'
map2['key2']='val2'
map2['key1']='value1'
map2.pop('key1')
print(type(map1),map1)
print(type(map2),map2)